src/js/game/LevelMVC/LevelView.js

```diff
@@ -1059,10 +1034,19 @@ module.exports = class LevelView {
     tween.start();
   }
 
+  positionToScreen(position, isOnBlock = false) {
```

Comment:
Worth a short function description/param tags?

Reply:
Feels a little strange that we take [x,y] (an array) as input, but then output {x, y} (an object).

Reply:
Yeah, I think I prefer `{x, y}` everywhere.  There are a ton of places in the code that currently use `[x, y]`.